sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner_test.py

```diff
@@ -240,6 +240,30 @@ def test_multimap_side_input(self):
               lambda k, d: (k, sorted(d[k])), beam.pvalue.AsMultiMap(side)),
           equal_to([('a', [1, 3]), ('b', [2])]))
 
+  def test_multimap_multiside_input(self):
```

Comment:
This test breaks Spark VR test: https://issues.apache.org/jira/browse/BEAM-9862. Please either support the same function for Spark or sickbay it.

Reply:
Thanks for reporting Boyuan, this was a flaw with the Spark runner. Fix: #11644